storage_broker/src/metrics.rs

```diff
@@ -23,3 +23,35 @@ pub static NUM_SUBS_ALL: Lazy<IntGauge> = Lazy::new(|| {
     )
     .expect("Failed to register metric")
 });
+
+pub static PROCESSED_MESSAGES_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "storage_broker_processed_messages_total",
+        "Number of messages received by storage broker, before routing and broadcasting"
+    )
+    .expect("Failed to register metric")
+});
+
+pub static BROADCASTED_MESSAGES_TOTAL: Lazy<IntCounter> = Lazy::new(|| {
```

Comment:
s/broadcasted/broadcast

Reply:
Why?

Reply:
well, 1) another metrics 'BROADCAST_DROPPED_MESSAGES_TOTAL' is called like that 2) broadcast conjugates without 'ed' :)

Reply:
Oh, irregular verb, I see.

` 1) another metrics ` is like "**broadcast** (process) dropped messages", and this metric is about "the total number of **broadcasted** messages (ever)".

Closing the thread :)